.github/workflows/docker.yml

```diff
@@ -1,4 +1,4 @@
-name: github-docker
```

Comment:
I think the `github` part was to separate these runs visually from the `Jenkins: ...` workflows. I dont feel particularly strongly about it, but wouldnt `Github: ...` be more similar to the other workflows?

Reply:
In my opinion the little icon is enough to identify github workflows -- at least on the website. On notification emails about failing CI, it might be still be nice to immediately see that a github runner failed.

I am open to rename them to `Github: <...>`. The main purpose of my suggestion was to reduce the length of the complete string that identifies a runner.